lib/help.ts

```diff
@@ -1,41 +1,11 @@
 import { IModify, IRead } from '@rocket.chat/apps-engine/definition/accessors';
-import { IRoom } from '@rocket.chat/apps-engine/definition/rooms';
-import { ISlashCommandPreviewItem, SlashCommandPreviewItemType } from '@rocket.chat/apps-engine/definition/slashcommands';
-import { IUser } from '@rocket.chat/apps-engine/definition/users';
-import { Commands } from '../commands/GitLabCommands';
+import { SlashCommandContext } from '@rocket.chat/apps-engine/definition/slashcommands';
 import { sendNotification } from './sendNotification';
 
-export function getHelpPreviewItems(): Array<ISlashCommandPreviewItem> {
-    return [{
-        id: 'setup',
-        type: SlashCommandPreviewItemType.TEXT,
-        value: 'Setup',
-    }, {
-        id: 'create',
-        type: SlashCommandPreviewItemType.TEXT,
-        value: 'Create',
-    }, {
-        id: 'search',
-        type: SlashCommandPreviewItemType.TEXT,
-        value: 'Search',
-    }];
-}
-
-export async function executeHelpPreviewItem(item: ISlashCommandPreviewItem, read: IRead, modify: IModify, sender: IUser, room: IRoom): Promise<void> {
-    let msg: string;
-    switch (item.id) {
-         case 'setup':
-             msg = 'Command usage: `/gitlab setup <your_auth_token>`';
-             break;
-        case 'create':
-            msg = 'Command usage: `/gitlab create <repository id/ path> <issue title> <description>`';
-            break;
-        case 'search':
-            msg = 'Command usage: `/gitlab search <projects/issues> <keyword>`';
-            break;
-        default:
-            msg = 'No usage for this command is assigned';
-            break;
-    }
-    await sendNotification(msg, read, modify, sender, room);
+export function help(context: SlashCommandContext, read: IRead, modify: IModify): void {
+    const msg = 'The commands I can understand' + '\n' +
+                'Usage: `/gitlab setup token <your_auth_token>`' + '\n' +
+                'Usage: `/gitlab create issue <repository id/ path> <issue title> <description>`' +  '\n' +
+                'Usage: `/gitlab search issues <keyword> (optional: <created-by-me>/<assigned-to-me> )`';
```

Comment:
Shouldn't we use a template string here? Especially with multiline literals this enhances readability IMO

Reply:
Sure.

Reply:
I thought of [Template strings](https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/template_strings). This could look like:

```
    const msg = `The commands I can understand:
\t\`/gitlab setup token <your_auth_token>\`
\t\`/gitlab create issue <repository id/ path> <issue title> <description>\`
\t\`/gitlab search issues <keyword> (optional: [created-by-me | assigned-to-me | all])\`;
```

Reply:
`\t` has no effect here. afaik I tried, isn't it?.

Reply:
yes, that might be the case.